rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/TableRecords.java

```diff
@@ -182,7 +188,16 @@ public static TableRecords buildRecords(TableMeta tmeta, ResultSet resultSet) th
                     field.setKeyType(KeyType.PrimaryKey);
                 }
                 field.setType(col.getDataType());
-                field.setValue(resultSet.getObject(i));
+                if (col.getDataType() == JDBCType.BLOB.getVendorTypeNumber()) {
```

Comment:
in fact，just oracle can run in this code. if db is mysql and the col type=blob, col.getDataType() will return LONGVARBINARY（-4）.